src/main/java/org/folio/circulation/domain/FeeFineRepository.java

```diff
@@ -0,0 +1,50 @@
+package org.folio.circulation.domain;
+
+import static org.folio.circulation.support.http.ResponseMapping.forwardOnFailure;
+import static org.folio.circulation.support.http.ResponseMapping.mapUsingJson;
+
+import java.util.concurrent.CompletableFuture;
+
+import org.folio.circulation.support.Clients;
+import org.folio.circulation.support.CollectionResourceClient;
+import org.folio.circulation.support.Result;
+import org.folio.circulation.support.http.client.CqlQuery;
+import org.folio.circulation.support.http.client.PageLimit;
+import org.folio.circulation.support.http.client.Response;
+import org.folio.circulation.support.http.client.ResponseInterpreter;
+
+import io.vertx.core.json.JsonObject;
+
+public class FeeFineRepository {
+  private static final Integer PAGE_LIMIT = 500;
```

Comment:
How was the limit of 500 fee / fines decided upon?

Reply:
Removed that.